src/client/assets/styles/components/_about.scss

```diff
@@ -32,8 +32,37 @@
     }
 
     &__tertiary-text {
-        margin-bottom: $default-spacing * 2;
+        margin-bottom: $default-spacing;
         color: $foreground-lighter;
         white-space: pre-line;
     }
-}
\ No newline at end of file
+
+    &__environment {
+        margin-top: $default-spacing;
+
+        select::-ms-expand {
+            display: none;
+        }
+    }
+
+    &__environment-text {
+        color: $foreground-light;
+    }
+
+    &__environment-select {
+        display: block;
+        margin: auto;
+        margin-top: 5px;
+        padding: 5px;
+        width: 60%;
+        outline: 0;
+        border: 1px solid grey;
+
+        /* arrow styling */
+        -webkit-appearance: none;
+        appearance: none;
+        background: linear-gradient(grey, grey) no-repeat, linear-gradient(-135deg, rgba(255, 255, 255, 0) 50%, white 50%) no-repeat, linear-gradient(-225deg, rgba(255, 255, 255, 0) 50%, white 50%) no-repeat, linear-gradient(black,grey) no-repeat;
```

Comment:
How would I collapse it further? I have multiple, different gradients, and I thought I already had them in short-hand notation. Would welcome any suggestions you have.